release/ray_release/glue.py

```diff
@@ -379,7 +382,9 @@ def run_release_test(
         buildkite_group(":mag: Interpreting results")
         # Only handle results if we didn't run into issues earlier
         try:
-            handle_result(test, result)
+            result_is_empty = handle_result(test, result)
+            if result_is_empty:
+                raise fetch_command_exception
```

Comment:
As stated above, I think it's enough if `handle_result` just raises an exception